azkaban-common/src/main/java/azkaban/metrics/ContainerizationMetricsImpl.java

```diff
@@ -185,6 +203,13 @@ public void sendCleanupStaleFlowHeartBeat() {
     cleanupStaleFlowHeartBeat.mark();
   }
 
+  @Override
+  public void markCleanupStaleStatusFlowNumber(String status, long n){
```

Comment:
do you think this function name is a little bit confusing? It says mark but its impl is to "add".

Reply:
good catch, let me update